src/index.js

```diff
@@ -1,10 +1,13 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+import App from "./App";
+import * as serviceWorker from "./serviceWorker";
+import { initSocket } from "./socket";
 
-ReactDOM.render(<App />, document.getElementById('root'));
+initSocket(() => {
```

Comment:
before the app is render, lets init the socket, the socket has to be init only once per session